.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Build
+
+on:
+  push:
```

Comment:
What is the value to having this build on push? I would suggest that we remove the push scheduler and just use the pull_request target

Reply:
This triggers the build whenever anybody pushes in main.
If the configuration permits pushes to main, even for Maintainers, I'll agree with you.
If otherwise, this build would signal a corrupting commit directly and enables quick fixing.